app/models/piece.rb

```diff
@@ -35,8 +35,6 @@ def check_mate?
   def valid_move?(new_row, new_column)
     return false if out_of_boundary?(new_row, new_column)
     return false if no_move?(new_row, new_column)
-    #return false unless proper_length?(new_row, new_column)
-    true
```

Comment:
@Ruby-Hot-Chili-Peppers/MightyPawn should "true" be here or in the specific piece model. I only bring this up because I think it was the opposite in the pawn_move branch. Thoughts?